src/code/game.c

```diff
@@ -9,6 +15,7 @@ ViMode sViMode;
 FaultClient sGameFaultClient;
 u16 sLastButtonPressed;
 
+#ifdef OOT_DEBUG
 void GameState_FaultPrint(void) {
```

Comment:
Could you also ifdef-out the declaraction in functions.h? I think that's what we decided on discord. There's at least these but here may be others:

GameState_FaultPrint
GameState_DrawInputDisplay
GameState_Alloc

Reply:
feel like `GameState_DrawInputDisplay` can be removed from functions.h entirely cause its not a very "public" function call that other things need access to. But this is a problem all over the codebase that needs to be addressed at some point, so we dont have to now

Reply:
I guess same with `GameState_FaultPrint`

Reply:
should I move them from functions.h to game.c? I'm not sure what's the convention here

Reply:
they dont need any declarations at all because they are defined before their usage. they can just be deleted from functions.h without changing anything else